e2e-tests/package.json

```diff
@@ -9,9 +9,11 @@
   "license": "ISC",
   "devDependencies": {
     "@playwright/test": "^1.42.1",
-    "@types/node": "^20.12.2"
+    "@types/node": "^20.12.2",
```

Comment:
This change seems unrelated.

Reply:
while updating e2e tests I noticed that the prettier config wasn't loading in, so I made this change. it is a bit unrelated but nice to have

Reply:
Could you please put this into a separate commit/pr?

Reply:
removed this from this pr